js/mage/adminhtml/browser.js

```diff
@@ -226,19 +223,13 @@ Mediabrowser.prototype = {
     },
 
     /**
-     * Return opener Window object if it exists, not closed and editor is active
-     *
-     * return object | null
+     * return object|null
      */
-    getMediaBrowserOpener: function() {
-         if (typeof(tinyMCE) != 'undefined'
-             && tinyMCE.get(this.targetElementId)
-             && typeof(tinyMceEditors) != 'undefined'
-             && ! tinyMceEditors.get(this.targetElementId).getMediaBrowserOpener().closed) {
-             return tinyMceEditors.get(this.targetElementId).getMediaBrowserOpener();
-         } else {
-             return null;
-         }
+    getMediaBrowserCallback: function() {
```

Comment:
well initially my idea was to have it in the `next` branch because, since all tinymce's variables/objects have different names between v3 and v6... everything is BC so having better names for our method didn't make any change.

later it was sad to have it into `main` which changes things a bit but at the end of the day, all of tinymce's scripts are still incompatible so... also if we don't change our naming... we don't get much compatibility anyway.

what do you think?